internal/cli/get.go

```diff
@@ -176,7 +176,7 @@ func (o *GetOptions) Run(ctx context.Context, args []string) error { //nolint:go
 	case kind == DeviceKind && len(name) > 0 && !o.Rendered:
 		response, err = c.ReadDeviceWithResponse(ctx, name)
 	case kind == DeviceKind && len(name) > 0 && o.Rendered:
-		response, err = c.GetRenderedDeviceSpecWithResponse(ctx, name, &api.GetRenderedDeviceSpecParams{})
+		response, err = c.GetRenderedDeviceWithResponse(ctx, name, &api.GetRenderedDeviceParams{})
```

Comment:
Currently the rendered output can only be printed as json or yaml.  Do we want to change this to simply print it as a single device?

Reply:
+1. The less we special-case the better.

Reply:
yeah going to need to be a followup

Reply:
https://issues.redhat.com/browse/EDM-1168